src/Header/RequestHandler.h

```diff
@@ -19,12 +19,19 @@
 class RequestHandler
 {
 public:
-    //instantiate instance with request, will parse to OperateData and check with Authenticator and call correspond api.
-    RequestHandler(const std::string &requestData);
+    //instantiate instance with accepted file descriptor, will receive request string and parse to OperateData, then check with Authenticator and call correspond api.
+    RequestHandler(const int &file_descriptor);
+    //receive request string and send to api
+    void RequestHandler::WaitForMessage();
+    //Get client file descriptor
+    int GetClient();
     //content of request
     std::unique_ptr<OperateData> data;
     //storing all feature api
     static const std::unordered_map<std::string, APIHandler&> apis;
+private:
+    //file descriptor
```

Comment:
(Optional) `File description for this request` to be clearer.

Reply:
Done